.drone.yml

```diff
@@ -8,7 +8,7 @@ volumes:
 
 steps:
 - name: restore-cache
-  image: &default-image maven:3.8.4-openjdk-11
+  image: maven:3.8.4-openjdk-11
```

Comment:
I am curious why maven 3.8.4 when our documentation says "maven >= 3.0.1 are installed".
https://checkstyle.org/beginning_development.html#Before_development

Reply:
I think this is the latest maven build for JDK11.

Any version of Maven >= 3.0 is suitable for build. But if the Maven version is older than the JDK, you need to additionally configure the toolchain.

Maven 3.8.x is recommended for JDK17+. 
Maven 3.6.x is recommended for JDK11+.